app/models/user.rb

```diff
@@ -38,6 +39,7 @@ class User < Common::RedisStore
 
   validates :uuid, presence: true
   validates :email, presence: true
+  validates :loa_current, presence: true
 
   # conditionally validate if user is LOA3
   with_options if: :loa3? do |user|
```

Comment:
is gender in the saml response yet? it's required by MVI so should probably be added to the conditional validation.